libs/server/Resp/BasicCommands.cs

```diff
@@ -53,6 +53,100 @@ bool NetworkGET<TGarnetApi>(ref TGarnetApi storageApi)
             return true;
         }
 
+        /// <summary>
+        /// GET
+        /// </summary>
+        bool NetworkGETEX<TGarnetApi>(ref TGarnetApi storageApi)
+            where TGarnetApi : IGarnetApi
+        {
+            if (parseState.Count < 1 || parseState.Count > 3)
+            {
+                return AbortWithWrongNumberOfArguments(nameof(RespCommand.GETEX));
+            }
+
+            var key = parseState.GetArgSliceByRef(0).SpanByte;
+
+            TimeSpan? expiry = null;
+            if (parseState.Count > 1)
+            {
+                var option = parseState.GetArgSliceByRef(1).SpanByte.AsSpan();
+                if (option.SequenceEqual(CmdStrings.EX))
+                {
+                    if (!parseState.TryGetLong(2, out var expireTime) || expireTime <= 0)
+                    {
```

Comment:
This logic is common for all EX|PX|EXAT|PXAT, can probably just do this if the option is not PERSIST and then do a switch assignment of expiry depending on the option?